build.sbt

```diff
@@ -17,6 +20,14 @@ lazy val newerCompilerLintSwitches = Seq(
   "-Ywarn-numeric-widen" // In 2.10 this produces a some strange spurious error
 )
 
+lazy val formattingPreferences = FormattingPreferences()
+  .setPreference(DanglingCloseParenthesis, Prevent)
+  .setPreference(DoubleIndentClassDeclaration, true)
```

Comment:
Could I interest you in adding
```
.setPreference(SpacesAroundMultiImports, true)
```
to stick closer to the scalariform defaults? (See https://github.com/sbt/sbt-scalariform/blob/master/src/main/scala/com/typesafe/sbt/SbtScalariform.scala#L43)